src/components/AddComment.jsx

```diff
@@ -0,0 +1,63 @@
+import { useContext, useState } from "react"
+import { postComment } from "../api"
+import { useParams } from "react-router-dom"
+import { UserContext } from "./Users";
+
+
+
+function AddComment({setIsLoadingComment}) {
```

Comment:
Things to add:
-Error message for unsuccessful posting of comment
-Stop users from being able to click submit twice in quick succession